docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -92,40 +94,41 @@ spec:
     ports:
       - containerPort: 443
 ```
-When a pod is attached to a certain network, it needs to set its "network" field:
+When a pod is attached to a certain network, it needs to set its "network" in annotations:
```

Comment:
you mean label?

Reply:
yes; thanks for pointing it out. will update